test/image/mocks/mapbox_bubbles.json

```diff
@@ -46,6 +46,29 @@
           "opacity": 0.5
```

Comment:
Looks (from the baseline image change) like this didn't actually work before, but now it does 🎉 
Was that a known bug?

Reply:
Yes (after some initial confusion) -> https://github.com/plotly/plotly.js/issues/2346